slf4j-jcl/pom.xml

```diff
@@ -21,33 +22,11 @@
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
     </dependency>
-
     <dependency>
       <groupId>commons-logging</groupId>
       <artifactId>commons-logging</artifactId>
-      <version>1.1.1</version>
+      <version>1.2</version>
```

Comment:
Not sure about this update. It appears more more than the version # needs to be performed. See #89 for discussion.